build.gradle

```diff
@@ -100,47 +98,42 @@ jacocoTestReport {
 }
 tasks.sonarqube.dependsOn tasks.jacocoTestReport
 
-def mainClass = 'com.iexec.worker.tee.pre.Main'
```

Comment:
Is there a reason why you remove this var and now repeat twice its value?

Reply:
it triggered a warning with a variable having the same name as a reserved keyword.
`mainClassName` did not work either and ti became complicated to find a meaningful name.
I followed the philosophy of `build.gradle` which seems to prefer those redundancy instead of variables.